app/main.py

```diff
@@ -1,4 +1,8 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
+    if not isinstance(dog_age, int) or not isinstance(cat_age, int):
+        raise ValueError
+    if cat_age < 0 or dog_age < 0:
+        raise ValueError
```

Comment:
Similar to the previous comment, include a descriptive message with the `ValueError` to indicate why the error is being raised, such as 'Age cannot be negative'.